index.php

```diff
@@ -167,6 +166,11 @@ public function checkForUpdate() {
 		$versionString = isset($response['versionstring']) ? $response['versionstring'] : '';
 
 		if ($version !== '' && $version !== $this->currentVersion) {
+			if (PHP_INT_SIZE < 8 && version_compare($version, '26.0.0', '>=')) {
```

Comment:
@blizzz is this version check correct? (not sure if $version contains e.g. 26.0.0.1 or 26.0.0)

Reply:
`version` is mapped to the internal version from the updater server.

https://github.com/nextcloud/updater_server/blob/d544e9db4098b77a4fcb0c9fd635a4f3cb24ccd7/src/Response.php#LL119C49-L119C64

https://github.com/nextcloud/updater_server/blob/d544e9db4098b77a4fcb0c9fd635a4f3cb24ccd7/config/config.php#L114

Reply:
okay, changed to 26.0.0.0